README.md

```diff
@@ -74,13 +79,16 @@ Running the Test Database Builder requires a Java Runtime version 11 or later.
 | Dependency                                                                          | Purpose                                                | License                          |
```

Comment:
I think you should separate the runtime dependencies to explain, under which conditions you need them.
Especially people should know that they have to add the MySQL dependency only in case they use the MySQL dialect. And that this is not a transitive dependency, meaning that they have to add it to their *own* POM.

Reply:
But the runtime dependencies are already separated